apps/web/src/components/forms/signin.tsx

```diff
@@ -54,6 +55,8 @@ export const SignInForm = ({ className }: SignInFormProps) => {
 
   const errorCode = searchParams?.get('error');
 
+  const loginRedirectPath = '/documents';
```

Comment:
What's the point of this one?

Reply:
Oh I see, move it to the top and make it shouty. This should probably be a real constant elsewhere in hindsight.

Reply:
I was thinking to have it here since in the near future we could check for redirects (eg https://localhost:3000/signin?redirectTo="/something") in the query param and attach it here

Reply:
Fair fair, I'd still say top of the file and shouty since it doesn't need to be reinstantiated when the component rerenders 😄